kingfisher_scrapy/spiders/colombia.py

```diff
@@ -29,14 +29,21 @@ class Colombia(LinksSpider):
         If ``from_date`` is provided and ``until_date`` don't, defaults to today.
     """
     name = 'colombia'
-    next_page_formatter = staticmethod(parameters('page'))
+    next_page_formatter = staticmethod(parameters('_id'))
     default_from_date = '2011-01-01'
 
+    @classmethod
+    def from_crawler(cls, crawler, *args, **kwargs):
+        spider = super(Colombia, cls).from_crawler(crawler, date_format='date', *args, **kwargs)
```

Comment:
Can just do `super()`, no? (We might want to clean this up globally in a new issue.)

`date_format` is 'date' by default.

```suggestion
        spider = super().from_crawler(crawler, *args, **kwargs)
```